app/src/main/java/com/example/themoviedb/loginvalidation/SharedPreferencesRepoImpl.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.themoviedb.loginvalidation
+
+import android.content.Context
+import com.example.themoviedb.application.MyApp
+import com.example.themoviedb.application.PREFS_NAME
+
+class SharedPreferencesRepoImpl : SharedPreferencesRepository {
+
+///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Remove this

Reply:
And override methods straight from ViewModel then?

Reply:
No I just meant this ///////////////////////////////////////////////////